### PR TITLE
Web proto endpoints

### PR DESCRIPTION
**Pull request by omo7765**

### CODE REVIEW

File: application/server.py [hunk 1]
```diff
@@ -66,6 +66,13 @@ def handle_create_room_form():
         confirmed_password = request.form.get('confirmed-room-password')
 
         if room_password == confirmed_password:
+<<<<<<< HEAD
```
Comment (yxd3549) [inline application/server.py]: Ummm. I think this is the result of a merge conflict. You'll need to solve this.

**Review by yxd3549 — CHANGES_REQUESTED** [review]
Good stuff! Just address the one comment regarding the merge conflict

File: application/server.py (hunk 1 above)
Comment (omo7765) [reply application/server.py]: i believe i fixed it now

**Comment by omo7765** [comment]
also did all of the connecting of web prototype. created a button for some pages that may need to be changed for game logic

**Review by yxd3549 — APPROVED** [review]
Great work!!

**Pull request merged**